manifests/init.pp

```diff
@@ -90,7 +90,12 @@
     name      => $service_name,
     enable    => true,
     hasstatus => false,
-    restart   => 'unbound-control reload',
+  }
+  if $control_enable {
+    Service[$service_name] {
+      restart   => 'unbound-control reload',
+    }
+    include unbound::remote
```

Comment:
This looks the same as whats in https://github.com/xaque208/puppet-unbound/pull/97